Enhance AttachmentImgCont styling for better responsiveness and aspect ratio support

adianasct opened the pull request:
Enhance AttachmentImgCont styling for better responsiveness and aspect ratio support

- Updated height calculation to handle cases where both width and height are provided, defaulting to 'auto' when applicable.
- Added aspect-ratio support to maintain image proportions based on provided width and height properties.

adianasct committed:
Enhance AttachmentImgCont styling for better responsiveness and aspect ratio support

- Updated height calculation to handle cases where both width and height are provided, defaulting to 'auto' when applicable.
- Added aspect-ratio support to maintain image proportions based on provided width and height properties.

karensct commented on src/components/Attachment/index.tsx:
```diff
@@ -959,7 +959,14 @@ const AttachmentImgCont = styled.div<{
+      : props.isRepliedMessage
+        ? '40px'
+        : props.width && props.height
+          ? 'auto'
```
you have added all images height auto, because our all images have width and height

for solve this I can suggest to use aspect ratio

adianasct committed:
Refactor AttachmentImgCont height calculation for improved responsiveness

- Simplified height logic to ensure proper rendering based on fitTheContainer, isRepliedMessage, and height properties.
- Maintained aspect-ratio support for better image proportion handling.

karensct committed:
Merge branch 'fix/disappearing-messages-hiden-issue' of https://github.com/sceyt/sceyt-chat-react-uikit into fix/image-resizing-issue

karensct committed:
Merge branch 'fix/disappearing-messages-hiden-issue' of https://github.com/sceyt/sceyt-chat-react-uikit into fix/image-resizing-issue

karensct committed:
Merge branch 'fix/disappearing-messages-hiden-issue' of https://github.com/sceyt/sceyt-chat-react-uikit into fix/image-resizing-issue

karensct committed:
Update aspect-ratio handling in AttachmentImgCont component

- Modified the aspect-ratio calculation to apply only when the container is not fitted and the message is not a replied message, improving image rendering behavior in various scenarios.
